spec/mongoid/criteria_spec.rb

```diff
@@ -3034,6 +3121,43 @@ class D
         end
       end
     end
+
+    context 'when plucking a localized field' do
+
+      before do
+        I18n.locale = :en
+        d = Dictionary.create(description: 'english-text')
+        I18n.locale = :de
+        d.reload.description = 'deutsch-text'
```

Comment:
:thought_balloon:  wondering why this `reload` here is needed. Do you need to reload document to reflect current locale?